src/libstd/env.rs

```diff
@@ -928,6 +939,16 @@ mod tests {
         set_var(&n, "VALUE");
         eq(var_os(&n), Some("VALUE"));
     }
+    
+    #[test]
+    fn test_env_args_reverse_iterator() {
+        let args: Vec<_> = args().rev().collect();
+        let l = args.len();
+        assert!(l >= 3);
+        assert!(args[l-1].contains("stdtest"));
+        assert_eq!(args[1], "--logfile");
+        assert!(args[0].ends_with(".log"));
+    }
```

Comment:
Could you add a unit test elsewhere with this test as this'd unfortunately break running the binary manually :(